src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/locale/create.yaml

```
@@ -0,0 +1,19 @@
+twig_hooks:
+    hooks:
+        'sylius_admin.locale.create.content':
+            form:
+                template: '@SyliusAdmin/shared/crud/create/content/form.html.twig'
+                configuration:
+                    render_rest: false
+
+        'sylius_admin.locale.create.content.form':
+            sections:
+                template: '@SyliusAdmin/locale/form/sections.html.twig'
+
+        'sylius_admin.locale.create.content.form.sections':
+            general:
+                template: '@SyliusAdmin/locale/form/sections/general.html.twig'
+
+        'sylius_admin.locale.create.content.form.sections.general':
+            code:
+                template: '@SyliusAdmin/locale/form/sections/general/code.html.twig'
```

I would be here for details:
```suggestion
        'sylius_admin.locale.create.content.form.sections':
            details:
                template: '@SyliusAdmin/locale/form/sections/details.html.twig'

        'sylius_admin.locale.create.content.form.sections.details':
            code:
                template: '@SyliusAdmin/locale/form/sections/general/code.html.twig'
```
but we could make it consistent in a separate PR